lib/modules/keyboardNav.js

```diff
@@ -1153,85 +1169,63 @@ function navigateTo(newWindow, thisHREF) {
 	}
 }
 
-const goModePanel = _.once(() => {
-	const $panel = $('<div id="goModePanel" class="RESDialogSmall">')
-		.append('<h3>Press a key to go:</h3><div id="goModeCloseButton" class="RESCloseButton">&times;</div>');
-
-	// add the keyboard shortcuts...
-	const $contents = $('<div class="RESDialogContents"></div>');
-	const $shortCutList = $('<table>');
-	for (const key in module.options) {
-		if (module.options[key].dependsOn === 'goMode') {
-			const niceKeyCode = getNiceKeyCode(key);
-			$shortCutList.append(`<tr><td>${niceKeyCode}</td><td class="arrow">&rarr;</td><td>${key}</td></tr>`);
+function handleGoModeCommand(func) {
+	if (module.options.useGoMode.value) {
+		if (goModeActive) {
+			func();
+			toggleGoMode();
 		}
+	} else {
+		func();
 	}
-	$contents.append($shortCutList);
-	$panel.append($contents);
-	$panel.on('click', '.RESCloseButton', module.goMode);
-	return $panel;
+}
+
+const goModePanel = _.once(() => {
+	const goModes = Object.keys(module.options)
+		.filter(key => module.options[key].goMode)
+		.map(key => ({ key, niceKeyCode: getNiceKeyCode(key) }));
+
+	return $(goModePanelTemplate({ goModes }))
+		.find('.RESCloseButton').click(toggleGoMode)
+		.end();
 });
 
-let goModeActive;
+let goModeActive = false;
 
-module.goMode = function() {
-	if (!module.options.useGoMode.value) {
-		return;
-	}
+function toggleGoMode() {
 	goModeActive = !goModeActive;
 	if (goModeActive) {
 		$('body').on('keyup', handleGoModeEscapeKey);
 		$(document.body).append(goModePanel().fadeIn());
```

Comment:
We need to change this to something like `goModePanel().appendTo(document.body).fadeIn()` because jQuery 3 no longer treats disconnected elements as hidden